code/modules/power/battery.dm

```diff
@@ -225,8 +225,8 @@ var/global/list/battery_online =	list(
 	var/data[0]
 	data["nameTag"] = name_tag
 	data["storedCapacity"] = round(100.0*charge/capacity, 0.1)
-	data["charge"] = charge ? charge/SMESRATE : 0 //Compensates for the input/output rate
-	data["capacity"] = capacity ? capacity/SMESRATE : 0
+	data["charge"] = charge ? charge/SMESRATE : 0; //Compensates for the input/output rate
+	data["capacity"] = capacity ? capacity/SMESRATE : 0;
```

Comment:
why are you adding EOL terminators?

Reply:
They existed a few lines down the code for a ? : check and I added them in there just in case, but I'll remove them.